### DISCUSSION TITLE
Does Kimai work with PostgreSQL?

### DISCUSSION THREAD

**Discussion by nooblag** [kimai/kimai#2221] [Questions & Answers]
Hi, I'm just wondering if Kimai works with PostgreSQL?

**Comment by kevinpapst** [comment]
Unfortunately not.

There was a demo branch, see https://github.com/kevinpapst/kimai2/pull/755 - but there was no feedback from the community and no one who wanted to help testing in 1,5 years.

**Reply by nooblag** [reply]
Okay, thanks, maybe one day!
